docker-compose.yml

```diff
@@ -24,6 +24,16 @@ x-quorum-def:
     - /bin/sh
     - -c
     - |
+      UDS_WAIT=10
+      for i in $$(seq 1 100)
+      do
+        if [ -S /qdata/tm/tm.ipc ];
+        then break
+        else
+          echo "Sleep $${UDS_WAIT} seconds to avoid race conditon for unix domain socket"
+          sleep $${UDS_WAIT}
+        fi
+      done
```

Comment:
Geth containers are configured with `restart: "on-failure"` so a Geth container keeps on trying until the socket file is available to connect. 

Is this logic required only for Windows?